releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -53,70 +53,116 @@
       defaultValue="Luna"
       storageURI="scope://Workspace"
       label="Target Platform">
-  <annotation
-      source="http://www.eclipse.org/oomph/setup/GlobalVariable"/>
-  <choice value="Photon"
-      label="Eclipse Photon- 4.8"/>
-  <choice value="Photon - Staging"
-      label="Eclipse Photon- 4.8 (Staging)"/>
-  <choice value="Oxygen"
-      label="Eclipse Oxygen - 4.7"/>
-  <choice value="Oxygen - Staging"
-      label="Eclipse Oxygen - 4.7 (Staging)"/>
-  <choice value="Neon"
-      label="Eclipse Neon - 4.6"/>
-  <choice value="Mars"
-      label="Eclipse Mars - 4.5"/>
-  <choice value="Luna"
-      label="Eclipse Luna - 4.4"/>
-  <choice value="Kepler"
-      label="Eclipse Kepler - 4.3"/>
-  <choice value="Juno"
-      label="Eclipse Juno - 4.2"/>
-  <choice value="Indigo"
-      label="Eclipse Indigo - 3.7"/>
-  <choice value="Helios"
-      label="Eclipse Helios - 3.6"/>
-  <choice value="Galileo"
-      label="Eclipse Galileo - 3.5"/>
-  <choice value="None"
-      label="None"/>
+    <annotation
+        source="http://www.eclipse.org/oomph/setup/GlobalVariable"/>
+    <choice
+        value="Photon"
+        label="Eclipse Photon- 4.8"/>
+    <choice
+        value="Photon - Staging"
+        label="Eclipse Photon- 4.8 (Staging)"/>
+    <choice
+        value="Oxygen"
+        label="Eclipse Oxygen - 4.7"/>
+    <choice
+        value="Oxygen - Staging"
+        label="Eclipse Oxygen - 4.7 (Staging)"/>
+    <choice
+        value="Neon"
+        label="Eclipse Neon - 4.6"/>
+    <choice
+        value="Mars"
+        label="Eclipse Mars - 4.5"/>
+    <choice
+        value="Luna"
+        label="Eclipse Luna - 4.4"/>
     <description>Choose the compatibility level of the target platform</description>
   </setupTask>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="xtext.nightly.composite"
-      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/latest/"/>
+      name="version.mwe2"
+      value="2.9.0"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.releases"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/releases"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.milestones"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/milestones"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.latest"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/latest"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext"
+      value="${p2.xtext.latest}"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.orbit"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit"/>
```

Comment:
wonder why we use this one and not a general orbit

Reply:
Likely also possible. Shall i change?

Reply:
would leave that for now. might have some reason